src/maxBy.js

```diff
@@ -1,24 +1,23 @@
-var _createMaxMinBy = require('./internal/_createMaxMinBy');
-var _curry2 = require('./internal/_curry2');
-var _gt = require('./internal/_gt');
+var _curry3 = require('./internal/_curry3');
 
 
 /**
- * Determines the largest of a list of items as determined by pairwise comparisons from the supplied comparator.
- * Note that this will return undefined if supplied an empty list.
+ * Takes a function and two values, and returns whichever value produces
+ * the larger result when passed to the provided function.
  *
  * @func
  * @memberOf R
- * @category Math
- * @sig (a -> Number) -> [a] -> a
- * @param {Function} keyFn A comparator function for elements in the list
- * @param {Array} list A list of comparable elements
- * @return {*} The greatest element in the list. `undefined` if the list is empty.
+ * @category Relation
+ * @sig Ord b => (a -> b) -> a -> a -> a
+ * @param {Function} f
+ * @param {*} a
+ * @param {*} b
+ * @return {*}
```

Comment:
I should write a parser to derive the JSDoc cruft from the type signature. The four lines above are nothing but noise.